### PR TITLE
[one-cmds] Add moving_average parameters

### PR DESCRIPTION
**Pull request by jinevening**
This adds parameters for moving_average in one-quantize.

ONE-DCO-1.0-Signed-off-by: Hyukjin Jeong <hj1.jeong@samsung.com>

---
Related to: https://github.com/Samsung/ONE/issues/11269
Draft PR: https://github.com/Samsung/ONE/pull/11409

### CODE REVIEW

File: compiler/one-cmds/one-quantize [hunk 1]
```diff
@@ -345,6 +361,31 @@ def _verify_arg(parser, args):
...
+            try:
+                moving_avg_batch = int(getattr(args, 'moving_avg_batch'))
+            except ValueError:
+                parser.error('moving_avg_batch must be integer')
+            try:
+                moving_avg_const = float(getattr(args, 'moving_avg_const'))
+            except ValueError:
+                parser.error('moving_avg_const must be float')
+        else:
+            parser.error('Unsupported mode')
```
Comment (jinevening) [inline compiler/one-cmds/one-quantize:387]: This check was added for negative test.

**Review by seanshpark — APPROVED** [review]
LGTM

File: compiler/one-cmds/one-quantize [hunk 2]
```diff
@@ -345,6 +361,31 @@ def _verify_arg(parser, args):
...
+            except ValueError:
+                parser.error('min_percentile must be float')
+            try:
+                max_percentile = float(getattr(args, 'max_percentile'))
+            except ValueError:
+                parser.error('max_percentile must be float')
+        elif getattr(args, 'mode') == 'moving_average':
+            # Check dtype
+            try:
+                moving_avg_batch = int(getattr(args, 'moving_avg_batch'))
```
Comment (glistening) [inline compiler/one-cmds/one-quantize:379]: `int` may not raise error for float number. You may use `isinstance(..., int)`.

File: compiler/one-cmds/one-quantize [hunk 3]
```diff
@@ -345,6 +361,31 @@ def _verify_arg(parser, args):
...
+            except ValueError:
+                parser.error('moving_avg_batch must be integer')
+            try:
+                moving_avg_const = float(getattr(args, 'moving_avg_const'))
```
Comment (glistening) [inline compiler/one-cmds/one-quantize:383]: Also, I am not sure `float` will raise error when `moving_avg_const` is integer.

File: compiler/one-cmds/one-quantize (hunk 2 above)
Comment (jinevening) [reply compiler/one-cmds/one-quantize:379]: The type of argument is string, so `int` raises an exception.

```
>>> int('0.1')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: invalid literal for int() with base 10: '0.1'
```

You might mean

```
>>> int(0.1)
0
```

File: compiler/one-cmds/one-quantize (hunk 3 above)
Comment (jinevening) [reply compiler/one-cmds/one-quantize:383]: Integer value is ok for `moving_avg_const`.

File: compiler/one-cmds/one-quantize (hunk 2 above)
Comment (jinevening) [reply compiler/one-cmds/one-quantize:379]: The negative test case in this PR tests if the exception is raised.

File: compiler/one-cmds/one-quantize (hunk 3 above)
Comment (jinevening) [reply compiler/one-cmds/one-quantize:383]: Hmm.. there is no check for `moving_avg_const` has to be [0,1]. Let me add the check in `record-minmax` in the next PR.

File: compiler/one-cmds/one-quantize (hunk 2 above)
Comment (glistening) [reply compiler/one-cmds/one-quantize:379]: Ah, the argument type is `string`! Thank you for explanation.

File: compiler/one-cmds/one-quantize (hunk 3 above)
Comment (glistening) [reply compiler/one-cmds/one-quantize:383]: I see. By `float`, any number is acceptable (including `int`). But it is okay. You use `float` to check whether it is number.

**Review by glistening — APPROVED** [review]
LGTM

**Pull request merged**